### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by BerePendzialek**

### CODE REVIEW

File: src/components/FavoritesPage/FavoritesPage.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import Icon from 'supercons'
+import styled from 'styled-components/macro'
+import Header from '../Header'
+
+export default function FavoritesPage() {
+  return (
+    <div>
```
Comment (badabam) [inline src/components/FavoritesPage/FavoritesPage.js:7]: Maybe this should be a `section`?

File: src/components/Header/Header.stories.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import Header from './Header'
+
+export default {
+  title: 'Components',
```
Comment (badabam) [inline src/components/Header/Header.stories.js:4]: Not sure what Storybook makes out of calling multiple stories `Components`, but we usually set the title to the component title e.g. `title: 'Components/Header'`

File: src/components/Header/Header.stories.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  component: Header,
+}
+
+const DefaultHeader = args => <Header {...args}> Favorites </Header>
```
Comment (badabam) [inline src/components/Header/Header.stories.js:8]: It's fine as is, but it's more common to call this const `Template`.

File: src/components/HomePage/HomePage.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components/macro'
+import { NavLink } from 'react-router-dom'
+import Icon from 'supercons'
+import Button from '../Button'
+import Header from '../Header'
+
+export default function HomePage() {
+  return (
+    <div>
```
Comment (badabam) [inline src/components/HomePage/HomePage.js:9]: Maybe a `section`?

File: src/components/Navigation/Navigation.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    <NaviWrapper>
+      <NaviBar>
+        <NaviLink exact to="/">
+          <Icon glyph="home" size={24} />
+        </NaviLink>
+        <NaviLink exact to="/favorites" data-testid="favorites">
+          <Icon glyph="like" size={24} />
+        </NaviLink>
+      </NaviBar>
+    </NaviWrapper>
```
Comment (badabam) [inline src/components/Navigation/Navigation.js:16]: As you are already in the `Navigation` component and this styled-components are scoped to it, you could get rid of the `Navi` prefix of all coponent names and make it more readable:
```suggestion
    <Wrapper>
      <Bar>
        <Link exact to="/">
          <Icon glyph="home" size={24} />
        </Link>
        <Link exact to="/favorites" data-testid="favorites">
          <Icon glyph="like" size={24} />
        </Link>
      </Bar>
    </Wrapper>
```

Comment (badabam) [reply src/components/Navigation/Navigation.js:16]: Also: you should be able to merge `NaviBar` and `Wrapper` into one element with one styling.

File: src/components/Playlist/Playlist.js [hunk 6]
```diff
@@ -31,12 +31,18 @@ export default function Playlist({
...
+`
 
 const WarmupWrapper = styled.div`
   display: grid;
 `
 
-const IntervalsTWrapper = styled.div``
+const IntervalsTWrapper = styled.div`
+  display: grid;
+`
```
Comment (badabam) [inline src/components/Playlist/Playlist.js:44]: If all your wrappers just do `display: grid`, you could reuse the same wrapper and don't need multiple ones.

**Review by badabam — APPROVED** [review]
Hi Berenice,
I found some possible improvements. Good job overall!

File: src/components/Header/Header.stories.js (hunk 2 above)
Comment (BerePendzialek) [reply src/components/Header/Header.stories.js:4]: You are right. I will unify the names. Somehow I started writing it so and keep on going.

File: src/components/HomePage/HomePage.js (hunk 4 above)
Comment (BerePendzialek) [reply src/components/HomePage/HomePage.js:9]: yes, it is better.

File: src/components/Navigation/Navigation.js (hunk 5 above)
Comment (BerePendzialek) [reply src/components/Navigation/Navigation.js:16]: Roger!

Comment (BerePendzialek) [reply src/components/Navigation/Navigation.js:16]: Roger!

**Pull request merged**